data/pools/transactionPool.go

```diff
@@ -990,10 +1006,27 @@ func (pool *TransactionPool) AssembleDevModeBlock() (assembled *ledgercore.Valid
 	defer pool.mu.Unlock()
 
 	// drop the current block evaluator and start with a new one.
+	pool.devMode = true
 	pool.recomputeBlockEvaluator(nil, 0)
 
 	// The above was already pregenerating the entire block,
 	// so there won't be any waiting on this call.
 	assembled, err = pool.AssembleBlock(pool.pendingBlockEvaluator.Round(), time.Now().Add(pool.proposalAssemblyTime))
 	return
 }
+
+// SetBlockTimeStampOffset sets the timestamp offset.
+// Only available in dev mode
+func (pool *TransactionPool) SetBlockTimeStampOffset(offset int64) {
+	pool.mu.Lock()
+	defer pool.mu.Unlock()
+	pool.devModeTimeStampOffset = offset
+}
+
+// GetBlockTimeStampOffset gets the timestamp offset.
+// Only available in dev mode
+func (pool *TransactionPool) GetBlockTimeStampOffset() int64 {
+	pool.mu.Lock()
+	defer pool.mu.Unlock()
+	return pool.devModeTimeStampOffset
+}
```

Comment:
This also needs to return a pointer in order for the "nil is unset" logic to work.

Reply:
resolving - outdated by newest changes